Mathlib/Algebra/Order/Group/Equiv.lean

```diff
@@ -0,0 +1,47 @@
+/-
+Copyright (c) 2025 Weiyi Wang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Weiyi Wang
+-/
+import Mathlib.Algebra.Group.Equiv.Defs
+import Mathlib.Algebra.Order.Group.Synonym
+
+/-!
+# Add/Mul equivalence for order type synonyms
+-/
+
+variable (α : Type*)
+
+/-- `toLex` as a `MulEquiv`. -/
+@[to_additive "`toLex` as a `AddEquiv`."]
```

Comment:
```suggestion
@[to_additive (attr := simps toEquiv) "`toLex` as a `AddEquiv`."]
```

Reply:
This produces `toLexMulEquiv_toEquiv`, which is basically renaming `toEquiv_toLexMulEquiv`. (`toEquiv_toLexMulEquiv` was introduced in #22420). This is where I don't know what our naming convention is and I found both examples in the library. `toEquiv` can be seen as a function application so the function name should appear first; but there are also example where a dot member is written the other way.

Reply:
Got it. I have applied the change and removed `toEquiv_toLexMulEquiv`

Reply:
Welp, the simp normal form checker rejected this
```
-- Mathlib.Algebra.Order.Group.Equiv
Error: /home/lean/actions-runner/_work/mathlib4/mathlib4/pr-branch/Mathlib/Algebra/Order/Group/Equiv.lean:16:24: error: toLexMulEquiv_toEquiv Left-hand side simplifies from
  (toLexMulEquiv α).toEquiv
to
  ↑(toLexMulEquiv α)
using
  dsimp only [*, @MulEquiv.toEquiv_eq_coe]
Try to change the left-hand side to the simplified term!
```

Reply:
Ugh, I'm so sorry my suggestion depends on #21031